### PR TITLE
Added a note to fill PR form in lesson 05

### PR DESCRIPTION
**Pull request by Haimantika**
## Description

This solves issue https://github.com/open-sauced/intro/issues/140 , where I added a note to fill PR form in lesson 05

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
Closes https://github.com/open-sauced/intro/issues/140

## Mobile & Desktop Screenshots/Recordings

NA

## Steps to QA

NA

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by CBID2** [comment]
Hey @Haimantika. Can you add `Closes` next to the issue number in the `Related Tickets and Documents section`?

**Comment by adiati98** [comment]
@Haimantika I think your PR here consists the previous commits from your changes on PR #133.

Can you try to pull and merge the `main` from upstream to your working branch (`fix_lesson_5`)?
Thank you.

**Comment by Haimantika** [comment]
> git pull

Oh thanks, so nothing pending from my end, right?

File: 05-how-to-contribute-to-open-source.md [hunk 1]
```diff
@@ -170,6 +170,7 @@ You'll need to have these tools donwloaded and installed on your local machine:
     ```
 
 11. Go to your forked repository on GitHub and create a pull request with the title `feat: Add <@github-username> as a contributor`.
+   !> Your pull request will be marked as invalid and may be closed if the form is incomplete.
```
Comment (adiati98) [inline 05-how-to-contribute-to-open-source.md:173]: We want to apply the suggestion on the issue. So, can you please:

1. Update the wording in point 11.
2. Add space for the alert.

```suggestion
11. Go to your forked repository on GitHub. Create a pull request with the title `feat: Add <@github-username> as a contributor` and fill in all areas in the pull request form.

   !> Your pull request will be marked as invalid and may be closed if the form is incomplete.
```

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hi @Haimantika,
Thank you for your PR! 🙌

I left a review here for you to fix before we can approve your PR.
I also updated your branch. So, can you please pull your branch before working on the fix?

Let me know when it's fixed and ready for a review.
Thank you. 🙂

**Comment by Haimantika** [comment]
> Hi @Haimantika, Thank you for your PR! 🙌
> 
> I left a review here for you to fix before we can approve your PR. I also updated your branch. So, can you please pull your branch before working on the fix?
> 
> Let me know when it's fixed and ready for a review. Thank you. 🙂

Accepted changes.

**Review by adiati98 — APPROVED** [review]
Thanks, @Haimantika! 🙌

git merge GIF

**Review by BekahHW — APPROVED** [review]

**Pull request merged**